server.go

```diff
@@ -1849,6 +1849,22 @@ func (s *server) Start() error {
 			return nil
 		})
 
+		// If peers are specified as a config option, we'll add those peers first.
+		if len(s.cfg.AddPeers) != 0 {
+			for _, peerAddr := range s.cfg.AddPeers {
+				parsedPeerAddr, err := lncfg.ParseLNAddressString(peerAddr, strconv.Itoa(defaultPeerPort), net.ResolveTCPAddr)
```

Comment:
Should call `parseAddr` that's available in the `rpcserver.go` file to also support Tor peer addresses.

Reply:
Makes sense! lmk what you think of how I handled this. I split out some of ParseLNAddressString into a new function, since we needed part of what's in it -- validating the pubkey and converting it from a string.